prow/plugins/plugins.go

```diff
@@ -514,6 +514,25 @@ func (pa *PluginAgent) Config() *Configuration {
 	return pa.configuration
 }
 
+// ensureConfig ensure that the configuration
+// doesn't contain any nil value
+func ensureConfig(config *Configuration) error {
+	var errors []string
+	for _, configuration := range config.Plugins {
+		for _, plugin := range configuration {
+			if plugin == "heart" && config.Heart == nil {
+				errors = append(errors, "nil struct: heart")
+			}
```

Comment:
Im not sure if this is the right way, shall i just test `configuration == "heart` ?

Reply:
This should be good, `configuration` is a slice